README.md

```diff
@@ -65,8 +65,8 @@ Note that release branches might contain non-released patches.
 ## GHC compatibility
 |      | Linux | Windows | macOS | Clash (released) | Clash (development version)
 |------|-------|---------|-------|------------------|--------------------------
-| 8.6  | ✔️     | ✔️       | ✔️     | 1.0 - 1.6        | ✔️
-| 8.8  | ✔️     | ❌      | ✔️     | 1.0 - 1.6        | ✔️
+| 8.6  | ✔️     | ✔️       | ✔️     | 1.0 - 1.6        |  ️
+| 8.8  | ✔️     | ❌      | ✔️     | 1.0 - 1.6        |  ️
```

Comment:
Maybe put an :x: in the last column.
Or drop 8.6 and 8.8 from this table.

Reply:
Let's not drop them, not even after we release 1.10. We provide information on historical releases in this table. I do wonder why 1.8 isn't listed in the table, this seems like an accidental omission.

I agree a cross would be better.

Reply:
Also, for another PR, the footnotes for that table say:
> We use Stack to test on Windows/macOS. There is no resolver yet that uses GHC 9.6, so this remains untested.

This is no longer the case, so we should start testing already!

Reply:
@DigitalBrains1 Could you check out https://github.com/clash-lang/clash-compiler/pull/2665? 

I originally intended the table to be an easy way to view which historical Clash/GHC versions are compatible with one another to make plotting an upgrade path easier. That is to say, I agree with @DigitalBrains1, let's keep them there. I'll fix the notation though!